CMakeLists.txt

```diff
@@ -256,6 +256,7 @@ endif()
 
 opm_set_test_driver(${CMAKE_CURRENT_SOURCE_DIR}/tests/run-parallel-unitTest.sh "")
 
+add_compile_definitions(BOOST_BIND_GLOBAL_PLACEHOLDERS)
```

Comment:
minimum cmake is 3.10, add_compile_options appeared in 3.12. use add_definitions(-DBOOST_BIND_GLOBAL_PLACEHOLDERS)

Reply:
Thanks, I have updated the PR.